hummingbot/client/command/gateway_command.py

```diff
@@ -34,14 +31,9 @@
     native_tokens,
     upsert_connection
 )
+from hummingbot.core.utils.gateway_http_client import gateway_http_client
```

Comment:
We have just created a `hummingbot.core.gateway` package recently, this new module should be moved there.